src/template_helpers.rs

```diff
@@ -6,6 +6,7 @@ use handlebars::{
     handlebars_helper, Context, Handlebars, HelperDef, JsonTruthy, PathAndJson, RenderError,
     RenderErrorReason, Renderable, ScopedJson,
 };
+use markdown::Options;
```

Comment:
It may be confusing to import something named `Options` globally in [template_helpers.rs](https://github.com/sqlpage/SQLPage/pull/856/files#diff-9ec0c0b9c5d3fca1355e8046e996e5bfc68fd0f61cb76085b26749a5cd2c7b5f) that actually is specific to the markdown helper

Reply:
Yes, good catch. I was trying to restrain myself from creating a `markdown` submodule. I'll sort this out now.